Add content for data licence episode (#11)
esciencecenter-digital-skills/Lesson-FAIR-Data-Climate: _episodes/licenses.md

- Attribution (BY): Credit must be given to you, the creator.
- Share-Alike (SA): Adaptations must be shared under the same terms.
- Non-Commercial (NC): Only noncommercial use of your work is permitted.
- Non-Derivative (ND): No derivatives or adaptations of your work are permitted.

The six CC licences are combinations of these cornerstones.

> ## Note
> Without a licence you keep the copyright to yourself. This prohibits anyone from using the data for their own work.
{: .callout}

These are some points to consider when choosing a licence:

- A licence cannot be revoked once the data has been shared.
- As the origial author, you can change the licence for future publications.
- Sensivite data / embargo
- The less restrictions you add to your data, the more it can be re-used

Further reading:
- <https://www.dcc.ac.uk/guidance/how-guides/license-research-data>
- <https://www.openaire.eu/research-data-how-to-license/>

> > ## Excercise

Author: ```suggestion
> ## Discussion
```

>- When was the last time you published some data?
>- What licence did you distribute it under?
>- Where did you share the data?
{: .discussion}

Licences in Geoscience
----------------------

In general, data sources in geoscience such as CMIP5 and CMIP6 are available under FAIR licences. That is, you can freely use the data, but there are still some requirements. For example, if you base your research on CMIP6 data, you must give proper credit to the project if you deposit your research data somewhere. This is specified in the Terms of Use.

You can find the Terms of Use for some widely used projects here:

- CMIP6: <https://pcmdi.llnl.gov/CMIP6/TermsOfUse/TermsOfUse6-1.html>
- CMIP5: <https://pcmdi.llnl.gov/mips/cmip5/terms-of-use.html>
- CORDEX: <http://is-enes-data.github.io/cordex_terms_of_use.pdf>

> >## Exercise

Author: ```suggestion
>## Discussion
```

> >- Where can you find the licence for CMIP6 data? What about CMIP5?

Author: ```suggestion
```
That's mentioned just before the discussion box

Author: Perhaps add "Can you find additional terms of licences that are applicable to your situation?"

>- How do the terms of use for CMIP6 data differ from the Creative Commons licences?
>- Can you reshare the data?
>- Should you attribute the authors of the data? If so, how?
>- Can derived data be used commecially?
>- **FIXME**: check what the answer should be
{: .discussion}

Further reading:

> - https://gmd.copernicus.org/articles/11/3659/2018/ (Section 4)

Author: link doesn't work

Data ownership
--------------

Possible owners of the data:
- You
- Institute you work for
- Funding agency

You may be restricted by your institutions’ policies on licenses

Data ownership is defined in the data management plan

> ## Excercise
>Take a few minutes to find out whether your institute has a license policy.
>
>- What does it say?

> >- Who owns *your* data?

Author: What to do if your institution doesn't have a policy?

{: .discussion}

> ## Excercise

> >- Imagine you have downloaded some data from source X and applied operation Y on it.

Author: FIXME

>- Which data licence should you put on it when you distribute the data?
>- Which licence can you not put on it when you distribute the data?
{: .discussion}

What does the EU require?
-------------------------

The European Commission has fully embraced the FAIR principles. Horizon2020 already mandates open access to all scientific publications, and is now running a pilot from 2017 for research data to be open by default with possibilities to opt out.

How the data will be licenced is part of the data management plan. Specificically:

- How will the data be licensed to permit the widest re-use possible?
- When will the data be made available for re-use (i.e. are the data under embargo)?
- Are the data usable by third parties (i.e. are there restrictions)?

In all cases, the Commission encourages researchers to provide access to their data in the broadest sense. They encourage authors to retain their copyright, but grant adequate licences to publishers. They recommend the Creative Common licences as a useful legal tool for providing open access.

> Further reading:

Author: Perhaps we can create an exercise out of this? Let them read the document and find the info themselves? In my experience it is easier to memorize "where to find info" than to memorize the actual info

- <https://ec.europa.eu/research/participants/docs/h2020-funding-guide/cross-cutting-issues/open-access-data-management/open-access_en.htm>
- <https://ec.europa.eu/research/participants/data/ref/h2020/other/hi/oa-pilot/h2020-infograph-open-research-data_en.pdf>
- In case you are really interested, have a look at [p251-253 of the Horizon2020 Programme](https://ec.europa.eu/research/participants/data/ref/h2020/grants_manual/amga/h2020-amga_en.pdf)

How are licences applied to research data?
------------------------------------------

> Adding a licence to data is easy. You just need to mention or refer to the licence in the same place where you upload your data. Some repositories like Zenodo allow you to select a licence, and they make sure it is clear under which licence the data are available.

Author: ```suggestion
In principle, a licence is valid when it is mentioned or referred to in the same place where you upload your data. Some repositories like Zenodo allow you to select a licence, and they make sure it is clear under which licence the data are available.
```

You can apply a licence by one of these ways:

- Choosing a licence when you upload your data to a repository
- Referring to the licence on the webpage where the data are hosted
- Attaching the licence to the metadata

> - Adding a readme file / licence file to your data

Author: Note: some projects (e.g. CMIP6) explicitly require you to add the licence to the metadata.
